modules/va_forms/app/models/va_forms/form.rb

```diff
@@ -43,6 +43,17 @@ def self.old_search(search_term: nil)
       query
     end
 
+    def self.normalized_form_url(url)
+      url = url.starts_with?('http') ? url.gsub('http:', 'https:') : expanded_va_url(url)
+      Addressable::URI.parse(url).normalize.to_s
+    end
+
+    def self.expanded_va_url(url)
+      raise ArgumentError, 'url must start with ./va or ./medical' unless url.starts_with?('./va', './medical')
+
+      "#{FORM_BASE_URL}/vaforms/#{url.gsub('./', '')}" if url.starts_with?('./va') || url.starts_with?('./medical')
+    end
```

Comment:
These methods were moved from `VAForms::FormBuilder` so that the logic could be more easily shared by the `perform` instance method of that class and the `sidekiq_retries_exhausted_block`. (When I was testing, I found that the `sidekiq_retries_exhausted` block cannot utilize the instance methods of the Sidekiq job class, even if they are public methods, since the block does not hold a reference to the job instance.)